Controllers/ToppingsController.js

```diff
@@ -5,9 +5,34 @@ const ObjectId = require("mongodb").ObjectId;
 
 const getAllToppings = (req, res) => {
   DbService.connectToDb(async (db) => {
-    const toppings = await ToppingsService.getAllToppings(db);
-    res.json(toppings);
+    const result = await ToppingsService.getAllToppings(db);
+    let jsonRes = ResponsesService.successful()
+    jsonRes.data = result;
+    return res.json(jsonRes);
   });
 };
 
+const updateTopping = (req, res) => {
+  const data = {
+    id: ObjectId(req.body.id),
+    wins: req.body.wins,
+    battles: req.body.battles,
```

Comment:
is the idea here to send the specified number of wins and battles? Can't the API work that out rather than getting the front end to do the maths? Always better to do things like this on the API side instead of the FE when possible